### PR TITLE
refactor(orc8r): remove old northbound listSubscribers handler

### PR DESCRIPTION
**Pull request by christinewang5**
## Summary
* Remove old northbound listSubscribers handler
* Updated tests to reflect change

## Test Plan
* ran tests 
image

### CODE REVIEW

File: lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers_test.go [hunk 1]
```diff
@@ -2149,14 +1762,17 @@ func TestSubscriberPolicy(t *testing.T) {
 	tests.RunUnitTest(t, e, tc)
 
 	// Get all, updated subscriber matches the expected value
+	expected = subscriberModels.PaginatedSubscribers{TotalCount: 1, NextPageToken: "", Subscribers: map[string]*subscriberModels.Subscriber{
```
Comment (hcgatewood) [inline lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers_test.go]: Nit: move the fields of the PaginatedSubscribers each to their own line

**Review by hcgatewood — COMMENTED** [review]
Looks great, thanks for adding!

I think to finish it out let's go ahead and just use the original endpoint name (so use the new endpoint, but no "v2" in the name). So make the change in the Go code (and validate at the rest api as you have so far), then for the NMS:

1. Search for `v2\b` (or just `v2`) in `nms/packages/magmalte/`, and you'll want to replace the subscribersv2 names with subscribers
2. [Run NMS precommit tests](https://magma.github.io/magma/docs/next/nms/dev_testing#pre-commit-tests)
3. [Spin up your local NMS](https://magma.github.io/magma/docs/next/basics/quick_start_guide#using-the-nms-ui) and sanity check (create a subscriber then delete it) the [subscribers page](https://magma.github.io/magma/docs/next/nms/subscriber)

cc @andreilee

File: lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml [hunk 2]
```diff
@@ -40,50 +40,6 @@ info:
...
-          required: true
-          schema:
-            $ref: '#/definitions/mutable_subscriber'
-      responses:
-        '201':
-          description: Success
-        default:
-          $ref: './orc8r-swagger-common.yml#/responses/UnexpectedError'
-
   /lte/{network_id}/subscribers_v2:
```
Comment (hcgatewood) [inline lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml]: Need to remove the `_v2` from this for the swagger ui

Screen Shot 2021-10-06 at 2 05 36 PM

**Review by hcgatewood — CHANGES_REQUESTED** [review]
Okay! As discussed offline, I pulled down your PR and validated the NMS changes. Everything lgtm!

Let's make that 1 swagger.v1.yml change (you'll have to build.py --generate again), then I'm ready to stamp and ship 🙂 

Screen Shot 2021-10-06 at 2 05 21 PM

**Comment by hcgatewood** [comment]
Side notes:

1. To pass the semantic pr check, prepend `refactor: ` to your PR title
2. You'll have to pull and rebase onto latest master to reconcile the merge conflict in lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers_test.go

File: lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml (hunk 2 above)
Comment (christinewang5) [reply lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml]: removed!

image

**Comment by christinewang5** [comment]
closing because there is a new pr with the correct rebasing: 
https://github.com/magma/magma/pull/9567

**Pull request closed without merging**